google-api-client-android/src/main/java/com/google/api/client/googleapis/extensions/android/accounts/GoogleAccountManager.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2012 Google Inc.
+ * Copyright 2012 Google Inc.
```

Comment:
Should they be `Google LLC` too?

Reply:
I asked the internal copyright folks about this a few months back in another project. They told me not to bother changing that, but new files should be LLC.